rclcpp_action/include/rclcpp_action/create_client.hpp

```diff
@@ -37,6 +37,7 @@ namespace rclcpp_action
  * \param[in] name The action name.
  * \param[in] group The action client will be added to this callback group.
  *   If `nullptr`, then the action client is added to the default callback group.
+ * \param[in] options options to pass to the underlying `rcl_action_client_t`.
```

Comment:
@fujitatomoya nit: capitalize `options`, as in
```suggestion
 * \param[in] options Options to pass to the underlying `rcl_action_client_t`.
```

Reply:
also some other places fixed too.